hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncRegionLocator.java

```diff
@@ -117,18 +123,29 @@ private boolean isMeta(TableName tableName) {
     }
   }
 
-  private List<String> getRegionName(RegionLocations locs) {
-    List<String> names = new ArrayList<>();
-    for (HRegionLocation loc : locs.getRegionLocations()) {
-      if (loc != null) {
-        names.add(loc.getRegion().getRegionNameAsString());
-      }
-    }
-    return names;
+  private static List<String> getRegionNames(RegionLocations locs) {
+    if (locs == null) { return Collections.emptyList(); }
```

Comment:
We still do not have an consensus on the style so let's keep the old way? At least for me I do not link to have this style of '{}' block, as if the line is too long, you still need to make a wrapping and then the code will be like
```
if () {
  xxx
}
```
but not
```
if () {
  xxx }
```
Seems not consistent...